File: exercicios/uma parada ai/cal.cpp
#include <iostream>

using namespace std;

int main() {
    char operador;
    double num1, num2;

    cout << "Digite um número: ";
    cin >> num1;

    cout << "Digite um operador (+, -, *, /): ";
    cin >> operador;

    cout << "Digite outro número: ";
    cin >> num2;

    switch (operador) {
        case '+':
            cout << "Resultado: " << num1 + num2 << endl;
            break;
        case '-':
            cout << "Resultado: " << num1 - num2 << endl;
            break;
        case '*':
            cout << "Resultado: " << num1 * num2 << endl;
            break;
        case '/':
            if (num2 != 0) {
                cout << "Resultado: " << num1 / num2 << endl;
            } else {
                cout << "Erro! Divisão por zero." << endl;
            }
            break;
        default:
            cout << "Operador inválido." << endl;
            break;
    }

    return 0;
}

File: chat/cal.java
import java.util.Scanner;

public class Calculadora {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        double num1, num2;
        char operador;
        double resultado;

        System.out.println("Digite o primeiro número:");
        num1 = scanner.nextDouble();

        System.out.println("Digite o segundo número:");
        num2 = scanner.nextDouble();

        System.out.println("Escolha a operação (+, -, *, /):");
        operador = scanner.next().charAt(0);

        switch (operador) {
            case '+':
                resultado = num1 + num2;
                System.out.println("Resultado: " + resultado);
                break;
            case '-':
                resultado = num1 - num2;
                System.out.println("Resultado: " + resultado);
                break;
            case '*':
                resultado = num1 * num2;
                System.out.println("Resultado: " + resultado);
                break;
            case '/':
                if (num2 != 0) {
                    resultado = num1 / num2;
                    System.out.println("Resultado: " + resultado);
                } else {
                    System.out.println("Não é possível dividir por zero.");
                }
                break;
            default:
                System.out.println("Operador inválido.");
        }

        scanner.close();
    }
}
